src/components/UI/Input/InputTypeThree.jsx

```diff
@@ -0,0 +1,30 @@
+import "./InputTypeThree.css";
+export const InputTypeThree = (props) => {
+	const {
+		wrapperClassName,
+		htmlFor,
+		labelClassName,
+		labelText,
+		type,
+		className,
+		placeholder,
+		name,
+		iconClassName,
+	} = props;
+	return (
+		<div className={wrapperClassName}>
+			<label htmlFor={htmlFor} className={labelClassName}>
+				{labelText}
+			</label>
+			<input
+				type={type}
+				className={className}
+				placeholder={placeholder}
+				required
+			/>
+			<button class="btn--icon">
+				<i class={iconClassName} />
+			</button>
+		</div>
```

Comment:
Why you are using button in input component

Reply:
The first two input component have opposite order of input and label tags and the third input have a button component.